ios/RNMagicScript/components/UiNodes/UiSliderNode.swift

```diff
@@ -141,26 +141,26 @@ import SceneKit
             self.height = height
         }
 
+        if let max = Convert.toCGFloat(props["max"]) {
```

Comment:
I think that swapping the min/max order is not enough. What if we have the following case:
```
<Slider min={-10} max={-2} />
```
?